requirements.txt

```
@@ -1,10 +1,10 @@
-hepunits==1.1.*
-particle==0.9.*
-decaylanguage==0.6.*;python_version>="3.4"
-awkward==0.12.*
+hepunits==1.2.*
+particle==0.11.*
+decaylanguage==0.7.*
+awkward>=0.12.0,<1.0
 uproot-methods==0.7.*
-uproot==3.11.*
-boost-histogram==0.6.*
-iminuit==1.3.*
+uproot==3.12.*
+boost-histogram==0.10.*
+iminuit==1.4.*
```

```suggestion
uproot~=3.12.0
boost-histogram~=0.10.0
iminuit~=1.4.0
```

Could also get `uproot-methods` above to be
```
uproot-methods~=0.7.0
```